packages/lib-classifier/src/plugins/drawingTools/components/Point/Point.stories.js

```diff
@@ -1,25 +1,143 @@
 import { withKnobs } from '@storybook/addon-knobs'
 import { storiesOf } from '@storybook/react'
+import React, { Component } from 'react'
+import { Box } from 'grommet'
+import { Provider } from 'mobx-react'
+import { Grommet } from 'grommet'
 import zooTheme from '@zooniverse/grommet-theme'
-import React, { Component, createRef, forwardRef } from 'react'
-import sinon from 'sinon'
-import { PointTool } from '@plugins/drawingTools/models/tools'
-import { Mark }from '@plugins/drawingTools/components'
-import Point from './Point'
+import asyncStates from '@zooniverse/async-states'
+import cuid from 'cuid'
+import SingleImageViewer from '@viewers/components/SingleImageViewer'
+import ClassificationStore from '@store/ClassificationStore'
+import SubjectViewerStore from '@store/SubjectViewerStore'
+import DrawingTask from '@plugins/tasks/DrawingTask/models/DrawingTask'
+import { DrawingTaskFactory, ProjectFactory, SubjectFactory, WorkflowFactory } from '@test/factories'
+
+const subject = SubjectFactory.build({
+  locations: [
+    { 'image/jpeg': 'http://placekitten.com/500/300' }
+  ]
+})
+
+const project = ProjectFactory.build()
+const workflow = WorkflowFactory.build()
+const drawingTaskSnapshot = DrawingTaskFactory.build({
+  instruction: 'Draw a point',
+  taskKey: 'T1',
+  tools: [{
+    color: zooTheme.global.colors['drawing-red'],
+    type: 'point'
+  }],
+  type: 'drawing'
+})
+
+const subTasksSnapshot = [
+  {
+    instruction: 'Name your favourite fruit.',
+    taskKey: 'T0.0',
```

Comment:
Is this the new way taskKeys are structured in the monorepo, as a decmial?

Reply:
Yes, check out [ADR 25](https://github.com/zooniverse/front-end-monorepo/blob/master/docs/arch/adr-25.md)